### ISSUE TITLE
 make optional sound and vibration when Time's up on 15-min count

### ISSUE THREAD

**Issue by CrisSevilleja** [BiologicalRecordsCentre/ABLE#467]
@kazlauskis @chrisvanswaay @DavidRoy Some volunteers mentioned if it would be possible to generate a sound when the time ends in the 15-min counts. Some others are happy with the vibration and wouldn't like the sound. Therefore I would provide the vibration and sound as an option on the app settings. 

could that be possible? what do you think?

**Comment by kazlauskis** [comment]
I agree this would be nice. We could show a little popup at the top with a message that the time is over, vibration and sound. Like a [push notification](https://duckduckgo.com/?q=push+notification&t=ha&va=j&iax=images&ia=images) which would also show up even when the device is locked.

**Comment by chrisvanswaay** [comment]
@kazlauskis I reported this earlier (but couldn't find the topic anymore, maybe it is closed), but last weekend noticed during a 15 min count that the 3 min warning does work, but still not the final closing warning (Android 14, google pixel 8, but this was also the case with my previous android 13 phone). I wondered if it would be an idea to do that last one (so the warning the count is over) 5 seconds before the end of the count (as I assume the reason it is not working is because the count is over). 5 seconds does not really make a difference, and it is irritating finding out you missed the end of the count. Maybe this could be tested fairly simple.
